packages/botonic-nlp/tests/tasks/ner/botonic-ner.test.ts

```diff
@@ -0,0 +1,157 @@
+import { join } from 'path'
+
+import { DatabaseManager } from '../../../src/embeddings/database-manager'
+import { DataLoader } from '../../../src/loaders/data-loader'
+import { Preprocessor } from '../../../src/preprocess/preprocessor'
+import { BotonicNer } from '../../../src/tasks/ner/botonic-ner'
+import { NerModelLoader } from '../../../src/tasks/ner/loaders/ner-model-loader'
+
+describe('Botonic NER', () => {
+  test('Loading model', async () => {
+    const ner = BotonicNer.from(
+      await NerModelLoader.from(join(__dirname, '..', '..', 'utils', 'model'))
+    )
+    expect(ner.model.name).toEqual('BiLstmNerModel')
+    expect(ner.locale).toEqual('en')
+    expect(ner.maxLength).toEqual(12)
+    expect(ner.entities).toEqual(['O', 'product', 'color', 'material', 'size'])
+    expect(ner.vocabulary).toEqual([
+      '<PAD>',
```

Comment:
too large (and hence difficult to maintain). I suggest checking the 1st 2 words and the length.